redis/src/cluster_client.rs

```diff
@@ -145,7 +145,10 @@ impl ClusterClientBuilder {
     /// Sets the default timeout for all new connections (default is None).
     ///
     /// If the value is `None`, then `get_connection` call may block indefinitely.
-    /// Passing Some(Duration::ZERO) to this method will result in an error.
+    ///
```

Comment:
These changes should be folded back into the commits that introduced them.

Reply:
Done